includes/emails/class-email.php

```diff
@@ -83,6 +83,32 @@ public static function init( $meta = [] ) {
 			$meta = apply_filters( 'hivepress/v1/emails/' . hp\get_class_name( $class ) . '/meta', $meta );
 		}
 
+		foreach ( hp\get_array_value( $meta, 'tokens', [] ) as $token ) {
+
+			// Get class object.
+			$class = hp\create_class_instance( 'HivePress\Models\\' . $token );
+
+			if ( ! $class ) {
+				continue;
+			}
+
+			// Get model fields.
+			$model_fields = $class->_get_fields();
+
+			if ( ! $model_fields ) {
+				continue;
+			}
+
+			foreach ( $model_fields as $field_name => $field_args ) {
+				if ( $field_args->get_arg( '_model' ) ) {
```

Comment:
Make sure that this is the only exception and other fields are always ok to display